modules/k8s/pod.go

```diff
@@ -137,5 +137,10 @@ func WaitUntilPodAvailableE(t testing.TestingT, options *KubectlOptions, podName
 
 // IsPodAvailable returns true if the pod is running.
 func IsPodAvailable(pod *corev1.Pod) bool {
-	return pod.Status.Phase == corev1.PodRunning
+	for _, containerStatus := range pod.Status.ContainerStatuses {
+		if !containerStatus.Ready && !*containerStatus.Started {
```

Comment:
Tests failed because it looks like `Started` is optional and not always set on the container status (and thus `nil`). Can you add a check for `nil`?

```
goroutine 60 [running]:
testing.tRunner.func1(0xc0003b9a00)
	/usr/local/go/src/testing/testing.go:874 +0x3a3
panic(0x17893c0, 0x299c1f0)
	/usr/local/go/src/runtime/panic.go:679 +0x1b2
github.com/gruntwork-io/terratest/modules/k8s.IsPodAvailable(0xc0009a4000, 0xc0003b9a00)
	/home/circleci/project/modules/k8s/pod.go:141 +0xbe
github.com/gruntwork-io/terratest/modules/k8s.WaitUntilPodAvailableE.func1(0x1d259a0, 0xc0003b9a00, 0x2, 0x1cdaba0)
	/home/circleci/project/modules/k8s/pod.go:124 +0x74
github.com/gruntwork-io/terratest/modules/retry.DoWithRetryE(0x1d259a0, 0xc0003b9a00, 0xc0003a5ce0, 0x29, 0x3c, 0x3b9aca00, 0xc0005f5d00, 0xc0003b9a00, 0x40be01, 0x17a7480, ...)
	/home/circleci/project/modules/retry/retry.go:74 +0x32c
github.com/gruntwork-io/terratest/modules/k8s.WaitUntilPodAvailableE(0x1d259a0, 0xc0003b9a00, 0xc00009abc0, 0x19c6de9, 0x9, 0x3c, 0x3b9aca00, 0x0, 0x0)
	/home/circleci/project/modules/k8s/pod.go:114 +0x15a
github.com/gruntwork-io/terratest/modules/k8s.WaitUntilPodAvailable(0x1d259a0, 0xc0003b9a00, 0xc00009abc0, 0x19c6de9, 0x9, 0x3c, 0x3b9aca00)
	/home/circleci/project/modules/k8s/pod.go:107 +0x77
github.com/gruntwork-io/terratest/modules/k8s.TestTunnelOpensAPortForwardTunnelToService(0xc0003b9a00)
	/home/circleci/project/modules/k8s/tunnel_test.go:59 +0x2eb
testing.tRunner(0xc0003b9a00, 0x1a6da08)
	/usr/local/go/src/testing/testing.go:909 +0xc9
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:960 +0x350
```

Reply:
oh sorry i missed it , i fixed in commit 120673a.